src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs

```diff
@@ -3,18 +3,25 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.Build.Framework;
+using Microsoft.DotNet.VersionTools.Automation;
+using Microsoft.DotNet.VersionTools.BuildManifest.Model;
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Xml.Linq;
 using MSBuild = Microsoft.Build.Utilities;
 
 namespace Microsoft.DotNet.Build.Tasks.Feed
 {
     public class PushToBlobFeed : MSBuild.Task
     {
+        private static readonly char[] ManifestDataPairSeparators = { ';' };
+        private const string DisableManifestPushConfigurationBlob = "disable-manifest-push";
+        private const string AssetsVirtualDir = "assets/";
```

Comment:
If constant, constant should be used in here as much as possible.

Reply:
There was a bit of confusion earlier about whether things should go in `assets/` or not, and what we want to pass to repos as `AssetsRootUrl`. (Currently we pass `.../assets/` to aspnet for runtime installers and `.../` to corefx/core-setup so they can download symbol packages.) IMO we should standardize and say all addressable flat assets should go in `assets/`, and since that's all we'll have, I planned to chop off the `assets/` part in the manifest because this name is well-known. I have this issue out to standardize the symbol package placement: https://github.com/dotnet/core-eng/issues/2274.